src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/LaplacianSmoothing.cpp

```diff
@@ -214,7 +214,7 @@ void LaplacianSmoothing::writeVTKFile(const QString& outputVtkFile)
   }
   Detail::ScopedFileMonitor vtkFileMonitor(vtkFile);
 
-  fprintf(vtkFile, "# vtk DataFile Version 2.0\n");
+  fprintf(vtkFile, " vtk DataFile Version 2.0\n");
```

Comment:
```suggestion
  fprintf(vtkFile, "# vtk DataFile Version 2.0\n");
```

@nyoungbq Did you mean to remove this one, I feel like this # should probably stay.